ccf_transaction_processor/CMakeLists.txt

```diff
@@ -24,8 +24,11 @@ include(${CCF_DIR}/cmake/ccf_app.cmake)
 # Error checking on environment variable
 # the PDO TP app will be signed with the RSA key whose locaation is
 # pointed to by PDO_ENCLAVE_CODE_SIGN_PEM
+IF (NOT DEFINED "$ENV{PDO_ENCLAVE_CODE_SIGN_PEM}")
```

Comment:
yes, thank you, i realized that this does not work. if you see, I commit removed this NOT DEFINED check in the lastest commit, just kept the NOT EXISTS check. (of course, i removed it since i couldn't quite figure out what the issue was, thank you for suggesting the alternative.)